src/plugin/regions/region.js

```diff
@@ -102,6 +102,7 @@ export class Region {
         }
         if (params.resize != null) {
             this.resize = Boolean(params.resize);
+            this.updateHandlesResize(this.resize);
```

Comment:
wondering if it would be better to use a javascript setter on `this.resize` to trigger the call to `updateHandlesResize`.  (@katspaugh are setters acceptable in this library?)

It seems like a user could very naturally do `region.resize = true` and expect it to work.  but maybe that's out of the scope of the PR?

Reply:
Hi, but in this case, if we'll use the setter for `resize`,  we should use it for all the region fields (instead of using `update` method). But in this case all the setters should invoke the `updateRender` method, and fire the `update` event. Correct?